app/questionnaire/dependencies.py

```diff
@@ -55,3 +52,28 @@ def get_block_ids_for_calculated_summary_dependencies(
             block_ids_by_section[key] = routing_path.block_ids
 
     return block_ids_by_section
+
+
+def map_calculated_summary_block_ids_to_routing_path(
```

Comment:
Similar to the above, this should be something like `get_routing_path_block_ids_by_section_for_calculated_summary_dependencies`. I don't think `block_ids` are necessary as if you iterate over a routing path, you would iterate over the block ids, so could also be `get_routing_paths_by_section_for_calculated_summary_dependencies`. Happy to keep block_ids bit for clarity. See what others think.

Reply:
Happy to go with `get_routing_path_block_ids_by_section_for_calculated_summary_dependencies` personally I like the clarity

Reply:
likewise 👍

Reply:
Updated to `get_routing_path_block_ids_by_section_for_calculated_summary_dependencies`